applications/crossbar/src/modules/cb_clicktocall.erl

```diff
@@ -447,6 +449,7 @@ build_originate_req(Contact, Context) ->
        ,{<<"Outbound-Callee-ID-Number">>, CalleeNumber}
        ,{<<"Outbound-Caller-ID-Name">>, FriendlyName}
        ,{<<"Outbound-Caller-ID-Number">>, OutboundNumber}
+	   ,{<<"Outbound-Call-ID">>, CallId}
```

Comment:
@emnvn 
Howdy :)

Setting the Outbound-Call-ID on the root of the originate request will cause it to fail if there are multiple endpoints (they will all attempt to use this call-id).  Instead a unique call id should be generate and set (same property name) on each of the 'Endpoints'.